config/vufind/RecordDataFormatter.ini

```diff
@@ -0,0 +1,45 @@
+; This file can be used to set global default options for fields, to add new fields and to configure fields
+; in the RecordDataFormatter.
+
+; This section is used to set global default option.
+[Global]
+; Enable or disable all fields by default (default is true)
+;enabled = false
+; Use change default renderMethod (default is "Simple")
```

Comment:
This comment doesn't quite make sense. Should we just remove the "Use change" part and start with "Default" or is there some other meaning that needs to be conveyed?

Reply:
fixed